static/app/views/issueDetails/groupReplays/groupReplays.tsx

```diff
@@ -64,7 +64,7 @@ function GroupReplays({group}: Props) {
     location,
     organization,
   });
-  const {allMobileProj} = useAllMobileProj();
+  const {allMobileProj} = useAllMobileProj({});
```

Comment:
nit: I would have made the arg optional so that the majority case doesn't need to include this empty options object.